src/Chapter2.hs

```diff
@@ -740,7 +770,9 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains num [x] = if (elem num x) then [x] else []
+contains num (x : xs) = if (elem num x) then x : (contains num xs) else contains num xs
```

Comment:
Try using the `filter` function here. It already allows keeping only those elements in the list, that satisfy the given predicate. So you won't need to implement recursion manually